src/_alert.css

```diff
@@ -0,0 +1,73 @@
+.fieldset
+{
+    width:100%;
```

Comment:
For spacing use the `rem` pattern as opposed to`%`, you can find more details about the spacing [here](https://github.com/rodgeraraujo/minus.css/blob/dev/src/_spacing.css).